bundle/edu.gemini.sp.vcs/src/test/scala/edu/gemini/sp/vcs/diff/MergeCorrectionSpec.scala

```diff
@@ -21,6 +21,8 @@ import Scalaz._
 class MergeCorrectionSpec extends Specification {
   import NodeDetail._
 
+  val lifespanId: LifespanId = LifespanId.random
```

Comment:
There is lifespan id per instance of a program.  So while a program exists in a database it uses the same lifespan id to mark edits made to that instance.  Here we're just checking that the version information is incremented so any lifespan id will do.